src/Main.java

```diff
@@ -15,52 +15,53 @@
     public static void main(String[] args) {
         System.out.println("Guess the number!");
         Random rand = new Random();
-        // Sample StringBuffer
-        StringBuffer sbuffer = new StringBuffer();
-        String playAgain = "Y";
+
         String newGame = "Y";
         if (newGame == "Y") {
             do {
-                int counter = 0;
+
                 int guess = rand.nextInt(100);
                 guess += 1;
-                EndOfGame eog = new EndOfGame(); //In order to use printMessage method you need to create an instance of EndOfGame
 
                 int playersguess = 0;

```

Comment:
In these day the camel case name is popular. You should use playersGuess instead of playersguess.